### PR TITLE
[photon-client] Styling fixes

### LINKED ISSUE #959: Multi-tag result section shows up even though pipeline isn't calibrated and multi-tag is disabled
image

### LINKED ISSUE #961: Show pipeline name in confirmation dialogues
We should show the name and the type of pipeline in the confirmation dialogues that pop up when deleting or duplicating a pipeline

### LINKED ISSUE #963: Streams not properly centering themselves on camera settings page
This issue is only prevalent on large displays where the stream's max-width comes into play

image

### LINKED ISSUE #964: Decimation value changes once calibration is started
The stream size changes here because the decimation value is changed to 2 once the calibration pipeline is enabled- ignoring the initially set value. Either the startPNPCalibration method should take decimation as a parameter or the selector for setting the decimation should be hidden until calibration starts

https://github.com/PhotonVision/photonvision/assets/68172138/aafd8d09-d501-48e3-8ec0-94a6fbd2e6b6

### PR DESCRIPTION
**Pull request by srimanachanta**
closes #964 
closes #963 
closes #961 
closes #959 

Also standardizes how units are shown in the tables
```
(angle)&deg;
(distance)&nbsp;m
```

### CODE REVIEW

**Comment by srimanachanta** [comment]
Need to test that the multitag section is correctly hidden. Create an apriltag pipeline and make sure it isn't calibrated and multitag is disabled, the section should be hidden

File: photon-client/src/components/dashboard/tabs/TargetsTab.vue [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
 import { PipelineType } from "@/types/PipelineTypes";
 import { useStateStore } from "@/stores/StateStore";
+
+const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings;
```
Comment (mcm001) [inline photon-client/src/components/dashboard/tabs/TargetsTab.vue:6]: This will magically update when the underlying settings do?

**Comment by mcm001** [comment]
image
Are the heights of these cards suppost to match?

**Comment by srimanachanta** [comment]
> image Are the heights of these cards suppost to match?

Nah

**Review by mcm001 — APPROVED** [review]

**Pull request merged**